test/e2e/test-apps/other/custom-tracing/src/main.js

```diff
@@ -0,0 +1,28 @@
+// eslint-disable-next-line import/order
+const { app } = require('electron');
+
+const Sentry = require('@sentry/electron');
+// eslint-disable-next-line no-unused-vars
+const Tracing = require('@sentry/tracing');
+
+Sentry.init({
+  dsn: '__DSN__',
+  debug: true,
+  autoSessionTracking: false,
+  tracesSampleRate: 1.0,
+  onFatalError: () => {},
+});
+
+async function doTransaction() {
+  const transaction = Sentry.startTransaction({ name: 'InitSequence', op: 'task' });
+  Sentry.configureScope((scope) => scope.setSpan(transaction));
```

Comment:
Why do we need to set the transaction on the scope?

Reply:
Ah, the docs say to do this but now I've read them again I see this is only important for linking them to errors!